setup.py

```diff
@@ -46,7 +40,8 @@ def run_tests(self):
 setup(
     name = 'datascience',
     packages = ['datascience'],
-    version = version,
+    python_requires='>=3.6',
+    version = __version__,
```

Comment:
Why change `version = version` to `version = __version__`?  This seems like it changes the logic of this code.  What is the purpose of this?

Reply:
This makes the build fail due to a circular dependency. I've reverted this logic now.